README.md

```diff
@@ -94,11 +99,17 @@ $ rails generate administrate:dashboard Foo
 ## Documentation
 
 To customize the appearance, behavior, and contents of the dashboard,
-see the guides at
-[https://administrate-prototype.herokuapp.com][prototype_heroku].
+we publish a set [of guides for the current release[released_docs].
```

Comment:
Is there a missing closing bracket `]` here?

Reply:
You are right!